executioner/languages/docker.ts

```diff
@@ -18,7 +18,7 @@ type File = {
   metadata?: any;
 };
 
-const DEFAULT_EXECUTION_TIMEOUT = parseInt(process.env.DEFAULT_EXECUTION_TIMEOUT || '60');
+const CPU_LIMIT_PER_EXECUTION = process.env.CPU_LIMIT_PER_EXECUTION;
```

Comment:
Can you name this as MAX_CPU_LIMIT_PER_EXECUTION, as the CP_LIMIT_PER_EXECUTION will come from the problem in the future

Reply:
We need to do some rework on how we execute and how we do isolation per test case execution